app/graphql/schema.graphql

```diff
@@ -695,6 +695,17 @@ type System implements Node {
     """
     profileId: String
   ): Int!
+  rulesFailedBySeverity(
```

Comment:
Ideally filtering parameters would be passed via the `filter` parameter, like with OS, or name, etc. and used by scoped search:
![Screenshot 2023-04-19 at 13 43 13](https://user-images.githubusercontent.com/7757/233064696-5c1e858d-82a0-4979-be04-92eba762a8db.png)